tools/c7n_mailer/c7n_mailer/record.py

```diff
@@ -21,6 +24,10 @@ def date_time_format(utc_str, tz_str='US/Eastern', format='%Y %b %d %H:%M %Z'):
     return parser.parse(utc_str).astimezone(gettz(tz_str)).strftime(format)
 
 
+def get_date_time_delta(delta):
```

Comment:
`
{% macro pretty_print_datetime(datetime_string, resource, action, policy, region) -%}
  {{ datetime_string|date_time_format(tz_str='US/Pacific', format='%Y %b %d %H:%M %Z') }}
{%- endmacro %}

{% macro get_maid_future_action(column_name, resource, action, policy, region) -%}
  {% for action in policy['actions'] %}
    {% if action['type'] == 'mark-for-op' %}
      {% set future_action_date = action['days']|get_date_time_delta() %}
      {{ action['op'] }}@{{ pretty_print_datetime(future_action_date) }}
    {% endif %}
  {% endfor %}
{%- endmacro %}
`

So now we can render any of the UTC strings from AWS's apis (creation dates, etc), and also render policy action day deltas, as a pretty date.